src/main/java/br/com/moip/models/Payment.java

```diff
@@ -0,0 +1,161 @@
+package br.com.moip.models;
+
+import br.com.moip.api.request.RequestMaker;
+import br.com.moip.api.request.RequestProperties;
+import br.com.moip.api.request.RequestPropertiesBuilder;
+import br.com.moip.exception.ValidationException;
+import br.com.moip.models.error.Errors;
+import br.com.moip.models.error.ErrorBuilder;
+import org.apache.http.entity.ContentType;
+
+import java.util.Map;
+
+import static br.com.moip.Moip.SANDBOX_URL;
+
+public class Payment {
+
+    private static final String ENDPOINT = "/v2/payments";
+    private static final String ENDPOINT_TO_PAY = "/v2/orders/%s/payments";
+    private static final String ENDPOINT_TO_AUTHORIZE = "/simulador/authorize";
+    private static final ContentType CONTENT_TYPE = ContentType.APPLICATION_JSON;
+    private RequestMaker requestMaker;
+
+    /**
+     * This method is used to create a payment. To make it, is necessary send the {@code body}
+     * filled with the payment data and the ID of the order that will be payed.
+     *
+     * @param   body
+     *          {@code Map<String, Object>} the request body.
+     *
+     * @param   orderId
+     *          {@code String} the Moip order external ID.
+     *
+     * @param   setup
+     *          {@code Setup} the setup object.
+     *
+     * @return  {@code Map<String, Object>}
+     */
+    public Map<String, Object> pay(Map<String, Object> body, String orderId, Setup setup) {
+        this.requestMaker = new RequestMaker(setup);
+        RequestProperties props = new RequestPropertiesBuilder()
+                .method("POST")
+                .endpoint(String.format(ENDPOINT_TO_PAY, orderId))
+                .body(body)
+                .type(Payment.class)
+                .contentType(CONTENT_TYPE)
+                .build();
+
+        return this.requestMaker.doRequest(props);
+    }
+
+    /**
+     * This method is used to capture a pre-authorized payment by its Moip payment external ID.
+     *
+     * @param   paymentId
+     *          {@code String} the Moip payment external ID.
+     *
+     * @param   setup
+     *          {@code Setup} the setup object.
+     *
+     * @return  {@code Map<String, Object>}
+     */
+    public Map<String, Object> capturePreAuthorized(String paymentId, Setup setup) {
+        this.requestMaker = new RequestMaker(setup);
+        RequestProperties props = new RequestPropertiesBuilder()
+                .method("POST")
+                .endpoint(String.format("%s/%s/capture", ENDPOINT, paymentId))
```

Comment:
Why is this endpoint declared here instead of a variable like the others?

Reply:
I will answer both comments about this here, okay?

If I had created a constant to set the value of these endpoints (`/capture` and `/void`), would be more two variables. With this constant I can use it more than one time and keep the standard of another classes.

Besides that, the other constants aren't `/v2/payments`.